R/getbb.R

```diff
@@ -26,67 +26,101 @@ bbox_to_string <- function(bbox) {
       bbox <- c (bbox["x", "coords.x1"], bbox["y", "coords.x1"], 
                  bbox["x", "coords.x2"], bbox["y", "coords.x2"])
     }
-    bbox <- paste0 (bbox[c(2,1,4,3)], collapse=",")
+    bbox <- paste0 (bbox[c(2, 1, 4, 3)], collapse = ",")
   } else {
     if (!is.null (names (bbox)) & 
         all (names (bbox) %in% c("left", "bottom", "right", "top"))) {
-      bbox <- paste0 (bbox[c ("bottom", "left", "top", "right")], collapse=",")
+      bbox <- paste0 (bbox[c ("bottom", "left", "top", "right")], collapse = ",")
     } else {
       x <- sort (bbox [c (1, 3)])
       y <- sort (bbox [c (2, 4)])
-      bbox <- paste0 (c (y [1], x[1], y [2], x [2]), collapse=",")
+      bbox <- paste0 (c (y [1], x[1], y [2], x [2]), collapse = ",")
     }
   }
   return(bbox)
 }
 
-#' Get bounding box for a given place name
+#' Get bounding box for a given place name.
 #' 
-#' Uses online services to convert a text string into a bounding box
+#' This function uses the free Nominatim API provided by OpenStreetMap to find
+#' the bounding box (bb) associated with place names.
+#' 
+#' It was inspired by the functions
+#' \code{bbox} from the \textbf{sp} package,
+#' \code{bb} from the \textbf{tmaptools} package and
```

Comment:
Maybe I've got a different version of LaTeX or something... Happy to go with `pkg` rather than **pkg** for the time being - may be worth looking at what more experienced developers use in their docs.  

In fact, I've just found the right solution: `\pkg{pkg}`: https://github.com/hadley/dplyr/blob/6153e136fa9397e88478fa6270d9d1f02eb5153e/R/data-lahman.r#L6